Marlin/src/lcd/ultralcd.cpp

```diff
@@ -2708,12 +2797,7 @@ void lcd_quick_feedback(const bool clear_buttons) {
     //
     // Auto Home
     //
-    MENU_ITEM(gcode, MSG_AUTO_HOME, PSTR("G28"));
-    #if ENABLED(INDIVIDUAL_AXIS_HOMING_MENU)
-      MENU_ITEM(gcode, MSG_AUTO_HOME_X, PSTR("G28 X"));
-      MENU_ITEM(gcode, MSG_AUTO_HOME_Y, PSTR("G28 Y"));
-      MENU_ITEM(gcode, MSG_AUTO_HOME_Z, PSTR("G28 Z"));
-    #endif
+    MENU_ITEM(submenu, MSG_LEVEL_BED_HOMING, lcd_homing_menu);
```

Comment:
Again, on most configurations there will only be a single item, so all this does is make Auto Home less accessible for most users.